2-With-broker/UserDetailsClient/UserDetailsClient/App.cs

```diff
@@ -14,6 +14,14 @@ public class App : Application
         public static string ClientID = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842";
         public const string BrokerRedirectUriOnIos = "msauth.com.yourcompany.UserDetailsClient://auth";
 
+        //The redirect uri on Android will need to be created based on the signature of the .APK used to sign it. 
+        //This means that it will be different depending on where this sample is run because Visual Studio creates
+        //a unique signing key for debugging purposes on every machine. You can figure out what that signature will be by running the following commands
+        //- For Windows: `keytool -exportcert -alias androiddebugkey -keystore %HOMEPATH%\.android\debug.keystore | openssl sha1 -binary | openssl base64`
+        //- For Mac: `keytool -exportcert -alias androiddebugkey -keystore ~/.android/debug.keystore | openssl sha1 -binary | openssl base64`
+        //For more details, please visit https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-net-use-brokers-with-xamarin-apps
+        public const string BrokerRedirectUriOnAndroid = "msauth://UserDetailsClient.Droid/{Your package signature}";
+
```

Comment:
@jennyf19 , @trwalke  - this redirect URI underlines 2 scenarios that I personally did not think of: 

1. What happens if developer does not set this kind of url but configures WithBroker(true)
2. Developer starts with WithBroker(false), AAD returns an auth code that tells MSAL they have to use the broker. Broker is on the device, but redirect URI is not correctly set (probably same as scenario 1, but worth checking)

Reply:
MSAL will throw an exception with the correct redirect URI so it makes getting around this super easy. 
2. the developer will have to change the redirect URI to the broker redirect URI. might be a good idea to call this out in documentation